cmd/minishift/cmd/image/export.go

```diff
@@ -69,7 +69,7 @@ func exportImage(cmd *cobra.Command, args []string) {
 	}
 	err = handler.ExportImages(imageCacheConfig)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Export of container images failed: %v", err))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Failed to export the container image: %v", err))
```

Comment:
This is an error message indicating something went wrong during the export of the configured image**s**. The concrete error is in _err_.